README.md

```diff
@@ -422,6 +422,24 @@ done manually by someone on the Azul team. Ask on Slack to find out more.
 
 [Terra Data Repository]: https://jade.datarepo-dev.broadinstitute.org/
 
+### 3.2.2 Personal account registration
+
+In order to register your personal email with SAM the following actions need to
+be performed:
+
+Ensure you are logged in with the correct Google Account:
+```
+gcloud auth login
+```
+For assistance refer to the Google authorization
+[documentation](https://cloud.google.com/sdk/docs/authorizing).
+
+Register your account with SAM using:
+```
+curl $AZUL_SAM_SERVICE_URL/register/user/v1  -d "" -H 
+"Authorization: Bearer $(gcloud auth --account $account_id print-access-token)"
+```
+
```

Comment:
Probably a good idea to include a reminder that the account must be then be authorized by an admin after being registered.